expo-maps/android/build.gradle

```diff
@@ -91,6 +91,9 @@ dependencies {
   implementation "com.google.android.gms:play-services-maps:18.0.2"
   implementation "com.google.maps.android:android-maps-utils:2.2.3"
   implementation "com.google.maps.android:maps-utils-ktx:3.3.0"
+  implementation "com.google.android.libraries.places:places:2.6.0"
   implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:${getKotlinVersion()}"
   implementation "org.jetbrains.kotlinx:kotlinx-coroutines-android:1.4.3"
+  //noinspection GradleDynamicVersion
+  implementation 'com.facebook.react:react-native:+'
```

Comment:
Is this safe option? 🤔

Reply:
Maybe we should specify version of RN here

Reply:
@szdziedzic  do you think it would be best to specify the exact version we use in our project?

Reply:
It shouldn't be specified. This package is coming from the local maven project - not from any online repository. 
So the version will be resolved based on the main project. That is a wanted behaviour here.

Reply:
Thanks!